collectors/python.d.plugin/postgres/postgres.chart.py

```diff
@@ -470,7 +479,7 @@
 QUERY_STANDBY_DELTA = {
     DEFAULT: """
 SELECT
-    application_name,
+    coalesce(prs.slot_name, psr.application_name) application_name,
```

Comment:
```suggestion
    COALESCE(prs.slot_name, psr.application_name) application_name,
```

Let's make the function uppercase